lib/takeaway.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'text'
+class Takeaway
+
+attr_reader :menu, :order, :text
+def initialize(menu=Menu.new, order=Order.new, text=Text.new)
+  @menu = menu.read
+  @order = order
+  @text = text
+end
+
+  def add_order(dish,number)
+    @order.add_order(dish,number)
+    print "#{number}x #{dish.to_s}(s) is added to your basket"
```

Comment:
You have `puts` and `prints` in many places in your app. That means that the app would hard to adapt to a different interface. Is there a way you can extract the printing to something else?